### ISSUE TITLE
Comment issue!

### ISSUE THREAD

**Issue by AbhiAgarwal** [seanjames777/SML-Language-Definition#17]
On my version I think this is an issue:

(\* (\*  *) *)

Just having a nested comment structure. It highlights the ones inside, but it doesn't highlight the far right one. This is valid in the language!

**Comment by seanjames777** [comment]
I'll look into it. It's almost certainly the comment matching regular expression.

**Comment by kavon** [comment]
yes, SML supports nested comments and I've been trying to ignore the coloring issue in sublime text for a while, so if you could fix this that would be great!
